docs/authorization/globus_app/apps.rst

```diff
@@ -0,0 +1,89 @@
+.. _globus_apps:
+
+.. currentmodule:: globus_sdk
+
+GlobusApps
+==========
+
+
+.. warning::
+
+    Currently ``GlobusApp`` only supports scripting-based use cases.
+
+    Hosted applications (i.e. web servers) should see :ref:`globus_authorizers` instead.
+
+
+:class:`GlobusApp` is a high level construct designed to simplify the process of
+of authenticating for Globus service clients.
```

Comment:
```suggestion
:class:`GlobusApp` is a high level construct designed to simplify authentication
for Globus service clients.
```

Reply:
I'm not sure why, but this rephrase is highlighting for me something else:
> Globus service clients.

"service clients" might mislead some folks into thinking that a "service client" is something meaningful vis-a-vis Globus Auth.

Perhaps...

> authentication for applications using globus-sdk clients.

?

Reply:
I'm hapyp to replace "Globus" with "globus-sdk" but I want to keep "service clients".

The term "client" is overloaded , particularly in this section of the docs so I want to be as clear as possible about what type of client we mean. I'll make the term "service client" be a link to the "[Service Clients](https://globus-sdk-python.readthedocs.io/en/stable/services/index.html)" section of our docs which hopefully will alleviate the concern that it's a Globus Auth thing.

Reply:
> The term "client" is overloaded

No contest!

:+1: for <code>globus-sdk :ref:&#96;service clients <service_clients>&#96;</code>

I'm not sure if we ever intentionally made "service client" a term of art -- other than it being the heading on that page, I don't think the term is used anywhere. And I would tend to pedantically call them "globus-sdk client classes" or "globus-sdk client objects" in most contexts. But I don't really object to the terminology and -- hey! -- it's the name used in the `services/index.rst` doc.

Reply:
Ended up going with `"globus-sdk services" where "services" is a link to our "service clients" page.

I also made an effort to qualify other usages of "client" across the page as "OAuth2 client"